docs/partial_source/deep_dive/ivy_frontends.rst

```diff
@@ -199,18 +205,51 @@ Ivy does not support the unique naming of individual operations, and so we omit
 .. code-block:: python
 
     # in ivy/functional/frontends/tensorflow/math.py
+    @to_ivy_arrays_and_back
     def tan(x, name=None):
         return ivy.tan(x)
 
 Likewise, :code:`tan` is also placed under :mod:`math`.
 By referring to the `tf.math.tan`_ documentation, we add the same arguments, and simply wrap :func:`ivy.tan` in this case.
 Again, we do not support the :code:`name` argument for the reasons outlined above.
 
+**NOTE**
+
+Many of the functions in the :mod:`tf.raw_ops` module have identical behaviuor to the functions that are generally in the TensorFlow namespace e.g :func:`tf.argmax`, with exception that the functions in the function in the :mod:`tf.raw_ops` module are specified to have key-word only arguments and in some case there is a naming difference of arguments.
```

Comment:
behaviuor - behaviour

Reply:
with exception that the functions in the function in the - with the exception that functions in the module, :mod:`tf.raw_ops`, are

Reply:
in some case there is a naming difference of arguments - in some cases there is a difference in argument naming